app/subscriber.py

```diff
@@ -43,7 +41,7 @@ def receipt_to_case(message: Message):
     try:
         payload = json.loads(message.data)  # parse metadata as JSON payload
         metadata = payload['metadata']
-        case_id, tx_id = metadata['case_id'], metadata['tx_id']
+        case_id, tx_id, questionnaire_id = metadata['case_id'], metadata['tx_id'], metadata['questionnaire_id']
```

Comment:
this line will error if case_id isn't in the payload, this was fixed in this pr: https://github.com/ONSdigital/census-rm-pubsub/pull/28

Reply:
done